flexmeasures/api/v3_0/assets.py

```diff
@@ -152,13 +152,28 @@ def index(
         if page is not None:
```

Comment:
I don't understand the comment above ("add search query for name"). Can it go?

Reply:
yes, it's a leftover comment